pkg/utils/k8sutils.go

```diff
@@ -110,3 +115,25 @@ func GetMD5Hash(text string) string {
 	hash := md5.Sum([]byte(text))
 	return hex.EncodeToString(hash[:])
 }
+
+var DelegateCSI = strings.ToLower(os.Getenv(CSIReconcileEnvVar)) == "delegate"
```

Comment:
Can we have this as a function to prevent code from changing the value from outside this package?

Reply:
done.